data/saas/config/outreach_config.yml

```diff
@@ -83,6 +86,26 @@ saas_config:
             - name: email
               identity: email
           data_path: data
+        delete:
+          method: POST
+          path: /api/v2/complianceRequests
+          param_values:
+            - name: requester_email
+              connector_param: requester_email
+            - name: email
+              identity: email
+          body: |
```

Comment:
In YAML the pipe `|` is used to indicate a "block scalar" value (multiline text). This allows us to denote JSON payloads inline with YAML while avoiding having to escape single or double quotes.

Reply:
that's cool, good thing i got to review this PR and learn something new :)